src/main/scala/mesosphere/marathon/core/deployment/DeploymentPlan.scala

```diff
@@ -186,6 +186,8 @@ case class DeploymentPlan(
       } else " NO STEPS"
     s"DeploymentPlan id=$id,$version$stepString\n"
   }
+
+  def targetIdsString = affectedRunSpecIds.mkString(", ")
```

Comment:
In this PR some occurrences of `plan.id` was replaced with `plan.targetIdsString` and some occurrences of `plan.target.id` was replaced with `plan.targetIdsString`.

`plan.target.id` is always `/` for RootGroup and we are only doing deployments on RootGroup afaik. On the other hand `plan.id` holds the UUID of the deployment and should be included in the log statements as well to make it searchable.

Reply:
What do you mean? I kept `plan.id` everywhere.

Reply:
... well, you are right. I`m sorry 🐼😪